content/ru/docs/tasks/administer-cluster/certificates.md

```diff
@@ -0,0 +1,280 @@
+             "key encipherment",
+             "server auth",
+             "client auth"
+           ],
+           "expiry": "8760h"
+         }
+       }
+     }
+   }
+   ```
+
+1. Создайте JSON-конфиг для запроса на подписание сертификата (CSR) (например,
+   `ca-csr.json`). Замените значения, помеченные угловыми скобками, на нужные.
+
+   ```json
+   {
+     "CN": "kubernetes",
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     },
+     "names":[{
+       "C": "<country>",
+       "ST": "<state>",
+       "L": "<city>",
+       "O": "<organization>",
+       "OU": "<organization unit>"
+     }]
+   }
+   ```
+
+1. Сгенерируйте ключ CA (`ca-key.pem`) и сертификат (`ca.pem`):
+
+   ```shell
+   ../cfssl gencert -initca ca-csr.json | ../cfssljson -bare ca
+   ```
+
+1. Создайте конфигурационный JSON-файл для генерации ключей и сертификатов для сервера API (например, `server-csr.json`). Замените значения, помеченные угловыми скобками, на нужные. `MASTER_CLUSTER_IP` – это cluster IP сервиса для сервера API, как описано в предыдущем подразделе. В приведенном ниже примере предполагается, что `cluster.local` используется в качестве доменного имени по умолчанию.
+
+   ```json
+   {
+     "CN": "kubernetes",
+     "hosts": [
+       "127.0.0.1",
+       "<MASTER_IP>",
+       "<MASTER_CLUSTER_IP>",
+       "kubernetes",
+       "kubernetes.default",
+       "kubernetes.default.svc",
+       "kubernetes.default.svc.cluster",
+       "kubernetes.default.svc.cluster.local"
+     ],
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     },
+     "names": [{
+       "C": "<country>",
+       "ST": "<state>",
+       "L": "<city>",
+       "O": "<organization>",
+       "OU": "<organization unit>"
+     }]
+   }
+   ```
+
+1. Сгенерируйте ключ и сертификат для сервера API (по умолчанию они сохраняются в файлах `server-key.pem` и `server.pem`, соответственно):
+
+   ```shell
+   ../cfssl gencert -ca=ca.pem -ca-key=ca-key.pem \
+        --config=ca-config.json -profile=kubernetes \
+        server-csr.json | ../cfssljson -bare server
+   ```
+
+## Распространение самоподписанного сертификата CA
+
+Клиентский узел может отказаться признавать самоподписанный сертификат CA действительным. В случае non-production-развертываний или развертываний, защищенных межсетевым экраном, самоподписанный сертификат CA можно распространить среди всех клиентов и обновить локальный список действительных сертификатов.
+
+Для этого на каждом клиенте выполните следующие операции:
+
+```shell
+sudo cp ca.crt /usr/local/share/ca-certificates/kubernetes.crt
+sudo update-ca-certificates
+```
+
+```none
+Updating certificates in /etc/ssl/certs...
+1 added, 0 removed; done.
+Running hooks in /etc/ca-certificates/update.d....
+done.
+```
+
+## API для сертификатов
+
+Для создания сертификатов x509, которые будут использоваться для аутентификации, можно воспользоваться API `certificates.k8s.io`  ([подробности](/docs/tasks/tls/managing-tls-in-a-cluster)).
```

Comment:
I would instead translate the link here literally, as it's in the original version now, so that the user will know (s)he is invited to the "Managing TLS in a cluster" task page.

Reply:
Great idea, thanks for the suggestion!